api/controllers/PersonalController.js

```diff
@@ -58,6 +57,19 @@ module.exports = {
       // probably save the model here
       // optionally flash a success message
       // then redirect
+      MedicalReport.create({
+        sin: req.body.social,
+        title: req.body.preferred_title,
+        first_name: req.body.first_name,
+        middle_name: req.body.middle_name,
+        last_name: req.body.last_name,
+        last_name_at_birth: req.body.birth_last_name,
+        birth_date: req.body.birthdate,
+        address: req.body.address,
+        telephone_number: req.body.telephone_number,
```

Comment:
Should be `telephone` to match 

![image](https://user-images.githubusercontent.com/1187115/71113281-3f11cc80-219b-11ea-8847-6f79169a16d3.png)

Reply:
Let's have a chat on making sure column names match input field names and related conventions.

Reply:
fixed the bug, will rationalize names later